java/shared/lang/message_digest_sha1_init.yml

```diff
@@ -0,0 +1,11 @@
+type: shared
+languages:
+  - java
+patterns:
+  - pattern: MessageDigest.getInstance($<JAVA_SHARED_LANG_MESSAGE_DIGEST_SHA1_INIT_ALGORITHM>)
```

Comment:
You could also match `java.security.MessageDigest` here, in case it's not imported but used with the full path instead.